_includes/tools/ava/install.md

```diff
@@ -0,0 +1,3 @@
+```console
+$ npm install --save-dev babel-core
```

Comment:
No, we're talking about the whole line. It's not necessary to install `babel-core`. AVA comes bundled with it.